pkg/network/ovn_kubernetes.go

```diff
@@ -193,7 +193,13 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 	// (which may not be a node in the cluster).
 	// However, this can never change, so we always prefer previous.
 	if sc.MTU == nil {
-		var mtu uint32 = uint32(hostMTU) - 100 // 100 byte geneve header
+		var mtu uint32 = uint32(hostMTU) - 50 // 50 byte geneve header
```

Comment:
I've seen the packets to be 1442 bytes with geneve. Looks like the geneve header is 58 bytes instead of 50.

Reply:
> I've seen the packets to be 1442 bytes with geneve. Looks like the geneve header is 58 bytes instead of 50.

ack, thanks Sai, I'll dig more into this and fix the number.

Reply:
You and Dan are right, did a packet capture and it sums upto 58 if there is nothing extra.